src/index.html

```diff
@@ -15,8 +15,105 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="index.html">
+        <img
+          class="logo"
+          src="images/logo.png"
+          alt="MOYO logo"
+        />
+      </a>
+
+      <nav>
+        <ul class="nav__list">
```

Comment:
You're using BEM elements correctly here, but `nav` block is not defined anywhere. Please define `nav` block for a parent HTML item.